lib/system/iterators.nim

```diff
@@ -136,7 +262,8 @@ iterator pairs*[T](a: openArray[T]): tuple[key: int, val: T] {.inline.} =
     yield (i, a[i])
     inc(i)
 
-iterator mpairs*[T](a: var openArray[T]): tuple[key: int, val: var T]{.inline.} =
+iterator mpairs*[T](a: var openArray[T]): tuple[key: int,
+    val: var T]{.inline.} =
```

Comment:
I don't think this Pr should format that, neither does it really add anything, nor is it related.

Reply:
I actually don't remember doing this :confused:. It must have been nimpretty.